xarray/tests/test_backends.py

```diff
@@ -626,35 +626,35 @@ def test_unsigned_roundtrip_mask_and_scale(self):
         encoded = create_encoded_unsigned_masked_scaled_data()
         with self.roundtrip(decoded) as actual:
             for k in decoded.variables:
-                self.assertEqual(decoded.variables[k].dtype,
-                                 actual.variables[k].dtype)
+                assert decoded.variables[k].dtype == \
```

Comment:
I prefer using parentheses to explicit line continuations (per PEP8), but I'm going to try to avoid being picky about it until we have automated tools that complain about it :)